articles/multifactor-authentication/developer/index.md

```diff
@@ -18,6 +18,10 @@ Apply the look-and-feel of your organization by working with our Guardian client
 * [Client library for Auth0 Guardian](https://github.com/auth0/auth0-guardian.js)
 * [Creating a Custom Guardian Widget](https://github.com/auth0/auth0-guardian.js/tree/master/example)
 
+## Enrollment email
```

Comment:
It is not kind of the opposite of what the body says? We are saying how not to send the emails through auth0.